### PR TITLE
when in a ssh session and in case of default user only display where …

### PR DESCRIPTION
**Pull request by ge045**
…instead of who@where:

This commit changes the prompt of the agnoster theme to only display the hostname instead of user@hostname in case the user is the default user and the session is ssh.

### CODE REVIEW

File: themes/agnoster.zsh-theme [hunk 1]
```diff
@@ -79,8 +79,10 @@ prompt_end() {
 
 # Context: user@hostname (who am I and where am I)
 prompt_context() {
-  if [[ "$USER" != "$DEFAULT_USER" || -n "$SSH_CLIENT" ]]; then
+	if [[ "$USER" != "$DEFAULT_USER" ]]; then
```
Comment (sachin21) [inline themes/agnoster.zsh-theme]: Indent is broken.
Could you change indents from 1 tab to 2 spaces?
FYI: https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide#indentation

**Comment by ge045** [comment]
@sachin21 yep, thanks. fixed it.

**Comment by sachin21** [comment]
+1

**Comment by ge045** [comment]
created pull request from master branch; will do a new request from feature branch

**Pull request closed without merging**